### ISSUE TITLE
The documentation of installing nvm is not working and out-dated.

### ISSUE THREAD

**Issue by ralf0131** [hexojs/site#392]
According to https://github.com/creationix/nvm, the latest way to install nvm should be as following:

Install script

To install or update nvm, you can use the install script using cURL:

```
curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
```

or Wget:

```
wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
```

**Comment by philippnagel** [comment]
#494 fixes this.

**Comment by NoahDragon** [comment]
Merged #494 , Thanks @philippnagel
